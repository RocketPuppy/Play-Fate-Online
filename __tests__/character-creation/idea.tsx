import * as React from "react";
import * as renderer from "react-test-renderer";
import Idea from "../../src/character-creation/idea";

test("renders all the pieces", () => {
  const render = renderer.create(<Idea />);

  expect(render.toJSON()).toMatchSnapshot();
});
