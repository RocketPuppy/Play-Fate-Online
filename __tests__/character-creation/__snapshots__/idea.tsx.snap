// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Character Idea
  </h2>
  <div>
    <label>
      <span>
        High Concept
      </span>
      <input
        onChange={[Function]}
        placeholder="Disciple of the Ivory Shroud"
        type="text"
      />
    </label>
    <label>
      <span>
        Trouble
      </span>
      <input
        onChange={[Function]}
        placeholder="The manners of a goat"
        type="text"
      />
    </label>
    <label>
      <span>
        Name
      </span>
      <input
        onChange={[Function]}
        placeholder="Landon"
        type="text"
      />
    </label>
  </div>
</div>
`;
